deploy/deploy-consumer.yaml

```
@@ -22,16 +22,13 @@ spec:
         app: rabbitmq-consumer
     spec:
       containers:
-      - name: rabbitmq-consumer
-        image: jeffhollan/rabbitmq-client:dev
-        imagePullPolicy: Always
-        command:
-          - receive
-        args:
-          - 'amqp://user:PASSWORD@rabbitmq.default.svc.cluster.local:5672'
-        envFrom:
-        - secretRef:
-            name: rabbitmq-consumer
+        - name: rabbitmq-consumer
+          image: jeffhollan/rabbitmq-client:dev
+          imagePullPolicy: Always
+          command:
+            - receive
+          args:
+            - "amqp://user:PASSWORD@rabbitmq.default.svc.cluster.local:5672"
```

Let's bring this back
```suggestion
            - "amqp://user:PASSWORD@rabbitmq.default.svc.cluster.local:5672"
        envFrom:
        - secretRef:	
            name: rabbitmq-consumer-secret
```

---

We are not using secretRef in consumer. Consumer is getting url from args. so this shouldn't be a problem